package.json

```diff
@@ -6,9 +6,9 @@
   "scripts": {
     "neg-test": "mocha --timeout 0 --bail --exit test/neg-http.mjs",
     "neg-test-ropsten": "mocha --timeout 0 --bail --exit test/neg-http.mjs",
-    "test-e2e-protocol": "mocha --timeout 0 --bail --parallel --exit test/e2e/protocol/*.test.mjs ",
+    "test-e2e-protocol": "LOG_LEVEL=error mocha --timeout 0 --bail --parallel --exit test/e2e/protocol/*.test.mjs ",
     "test-gas": "mocha --timeout 0 --bail --exit test/e2e/gas.test.mjs ",
-    "test-e2e-tokens": "mocha --timeout 0 --bail --exit test/e2e/tokens/*.test.mjs ",
+    "test-e2e-tokens": "LOG_LEVEL=error mocha --timeout 0 --bail --exit test/e2e/tokens/*.test.mjs ",
```

Comment:
Hey, currently mocha gets ENV variables from the command line, so that's why you see a bunch of

```js
if (process.env.VERBOSE) console.log("whatever")
```

I think adding this env variable in ```package.json``` will be confusing as we now have two "log levels" variables. I'd propose : 
1. either removing this LOG_LEVEL and starting the e2e process with ```LOG_LEVEL="error" npm run test-e2e-tokens", 
2. or changing ```package.json``` to ```VERBOSE=true LOG_LEVEL=error mocha etc etc etc```

I personally prefer using it the former way, because we don't need to change anything in the code if we want to change log levels, you just add or remove the env variable from ```npm``` cli command